### PR TITLE
content updated

### PR DESCRIPTION
**Pull request by strobba**
Content updated according to Daniel.

### CODE REVIEW

File: content/pages/medlemskap.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Odling
```
Comment (docNord) [inline content/pages/medlemskap.md]: Shouldn't title be "Medlemskap"?

File: content/pages/medlemskap.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+date: 2024-01-25
```
Comment (docNord) [inline content/pages/medlemskap.md:3]: 2024-03-09

File: content/pages/odling.md [hunk 3]
```diff
@@ -9,16 +9,33 @@ image:
...
+Gro Gamlestaden vill bedriva Stadsdelsodlingar i Östra Göteborg.
+
+Vi är just nu i samtal med kommunen angående möjliga platser där dessa planer
+kan realiseras.
 På dessa kvadratmeter har vi valt att odla basvaror åt samtliga
-gamlestadsbor, eller de människor som råkar ha vägarna förbi,  enligt ett
-växtföljdsschema om 4 år. Hur odlingen har planerats, vilka grödor som finns samt
+boende i stadsdelen, eller de människor som råkar ha vägarna förbi.
+Hur odlingen har planerats, vilka grödor som finns samt
 när dem är redo för skörd finns angivet på informationstavlan precis framför
```
Comment (docNord) [inline content/pages/odling.md]: Ska vara "de" inte "dem"

File: content/pages/odling.md [hunk 4]
```diff
@@ -9,16 +9,33 @@ image:
...
-odlingen. Vi ber alla som besöker området att hjälpa till att hålla det snyggt,
+odlingarna. Vi ber alla som besöker platserna att hjälpa till att hålla det snyggt,
 nyttja sunt förnuft samt sprida idéen om självförsörjande stadsdelar vidare till
```
Comment (docNord) [inline content/pages/odling.md]: idén

File: content/pages/odling.md [hunk 5]
```diff
@@ -9,16 +9,33 @@ image:
...
+
+Våra odlingar följer enkla principer.
+
+Vi ämnar att odla ekologiskt och hållbart utifrån ett längre perspektiv.
+Detta innebär att vi roterar våra grödor på ett sådant sätt att
+jorden inte utarmas på enstaka näringsämnen. Vi är också varse om vad
+vi planterar och hur det kan påverka vår omgivning.
+
+Våra grödor väljs utifrån hållpunkterna:
+högt matvärde, rikgivande,och enkla att sköta.
```
Comment (docNord) [inline content/pages/odling.md]: Högt matvärde, rikgivande och enkla att sköta.

**Review by docNord — COMMENTED** [review]
Nice!
I added a few comments on language and formalia. Good work, nice page!

**Comment by strobba** [comment]
Tack @docNord!

**Pull request merged**